packages/cloud-cognitive/src/components/AddSelect/AddSelectList.js

```diff
@@ -166,6 +167,11 @@ export let AddSelectList = ({
                   {item.children && (
                     <ChevronRight16 onClick={() => onNavigateItem(item)} />
                   )}
+                  {item.meta && (
+                    <div className={`${blockClass}-hidden-hover`}>
+                      <View16 onClick={() => setDisplayMetaPanel(item)} />
```

Comment:
This should ideally be an icon only button to support keyboard interactions. Something like 
```js
<Button
  renderIcon={View16}
  iconDescription={item.meta.title} // not sure if that's correct or not but something to pass to iconDescription since it is required for iconOnly buttons
  tooltipPosition="bottom"
  tooltipAlignment="center"
  hasIconOnly
  onClick={() => setDisplayMetaPanel(item)}
/>
```